src/main/java/core/basesyntax/service/impl/CsvParser.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dto.FruitTransaction;
+import core.basesyntax.models.Fruit;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.List;
+
+public class CsvParser implements FruitTransactionParser {
```

Comment:
I'd revert the naming or did something like this:
```suggestion
public class FruitTransactionParser implements TransactionParser<FruitTransaction> {
```
because both interface and and impl class sound concrete and interface name should be more abstract, here we have the ability to make some other `MeatTransaction` and `MeatTransactionParser` or even parametrize `TransactionParser` and its method to return `<T>` for example, to make it even more reusable in the future